src/commands/android/utils/adbWirelessConnect.ts

```diff
@@ -0,0 +1,73 @@
+import {Platform} from '../interfaces';
+import {execBinarySync} from './sdk';
+import Logger from '../../../logger';
+import inquirer from 'inquirer';
+import colors from 'ansi-colors';
+
+export async function connectAdbWirelessly(adbLocation: string, platform: Platform): Promise<boolean> {
+  try {
+    const deviceList = execBinarySync(adbLocation, 'adb', platform, 'devices -l');
+    if (deviceList !== null) {
+      if (!deviceList.toString().includes('transport_id')) {
+        Logger.log(colors.red('No devices found. Please connect a device and try again.'));
+
+        return false;
+      }
+    }
+
+    Logger.log(`${colors.bold('Follow the below steps to connect to your device wirelessly:\n')}
+  1. Enable USB debugging on your device.
+  2. Connect your device to your computer using a USB cable.
+  3. Grant permission for File transfer on your device.
+  4. Connect your device to the same network as your computer ${colors.grey('(you may connect your device to your computer\'s hotspot)')}.
+  5. Enable wireless debugging on your device.
+  6. Find the IP address, port number and pairing code of your device by going to 
+     ${colors.cyan('Settings > Wireless Debugging > Pair device with pairing code.')}\n`);
+
+    const deviceIPAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'deviceIP',
+      message: 'Enter the IP address of your device:'
+    });
+    const deviceIP = deviceIPAnswer.deviceIP;
```

Comment:
We should probably ask users to enter the whole thing here: `<ip_addr:port>` since they appear together on the device, and then split them ourselves on `:`. Also, give the format we are expecting in the question itself.

And in case the port is not provided (no second element after splitting with `:`), ask the port question as well.

Reply:
On a second thought, because there are two port numbers involved here, we can just keep the IP and port questions separate. Just add an instruction that IP is what comes before `:` and port is the number that comes after.